doc/api/fs.md

```diff
@@ -102,6 +102,11 @@ example `fs.readdirSync('c:\\')` can potentially return a different result than
 `fs.readdirSync('c:')`. For more information, see
 [this MSDN page][MSDN-Rel-Path].
 
+*Note:* On Windows, opening an existing hidden file using the `w` flag (either
+through `fs.open` or `fs.writeFile`) will fail with `EPERM`. Hidden files
+can be opened for writing with either `a` or `r+` flag. A call to
+`fs.truncate` can be used to reset the file contents. 
```

Comment:
You can't truncate a file opened in `'a'` mode.

```js
> var fd = fs.openSync('test.txt', 'a')
undefined
> fs.truncateSync(fd)
Error: EPERM: operation not permitted, ftruncate
```

Reply:
Hm, looks like on Windows you need to open the file with `r+` for `truncate` to work. That is unfortunate, there is no easy way to reproduce Linux behavior. One would need to call `truncate` with filename, ignore the error if the file does not exists, and then open the file with `a` parameter.

Reply:
I guess the best thing is to just document that?